CHANGES.rst

```diff
@@ -55,6 +55,7 @@ Improvements
   prevent non-admin users from turning off their profile's personal data
   synchronization (:pr:`5648`)
 - Add an option to sync event persons with users (:pr:`5677`)
+- Add color labels to the filter dropdown (:issue:`5675`, :pr:`5680`)
```

Comment:
```suggestion
- Add color labels to the editable list filter dropdown (:issue:`5675`, :pr:`5680`)
```

otherwise it's not really clear what this is about